homeassistant/components/mysensors/strings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "MySensors",
+  "config": {
+    "step": {
+      "confirm": {
```

Comment:
We need to add strings for the form data.

Reply:
Which form data?
Also, i now have created a dropdown with `vol.In("a","b")`. Is there a way to add strings for the options to the translations?

Reply:
The items we ask for input from the user in the config flow forms. Each item needs a string here. See the docs:
https://developers.home-assistant.io/docs/internationalization/core#config--options

Reply:
Items in a `vol.In` validator can not be translated. If we use the gateway type names here (Serial, TCP, MQTT) the need for translation is minimized.